setup.py

```diff
@@ -1,7 +1,6 @@
 from setuptools import setup, find_packages
 
 dependencies = [
-    'swsssdk>=1.3.0',
 ]
 
 test_deps = [
```

Comment:
>test_deps [](http://example.com/codeflow?start=0&length=9)

Do you need adding `swsssdk` into `test_deps`?

Reply:
Fixed.